Curtains/Curtains.Application/CurtainsService/EmailSender.cs

```diff
@@ -7,8 +7,8 @@ namespace Curtains.Application.CurtainsService
 {
     public class EmailSender : IMessageSender
     {
-        private const string _smtpClient = "smtp.gmail.com";
-        private const int _port = 587;
+        private const string _systemEmail = "zashtorim.noreply@gmail.com";
```

Comment:
it's should be in config
all

Reply:
and than if you need it get into const like this
`private readonly string _systemEmail = _config["EmailSettings:SystemEmail"]`

Reply:
and use it

Reply:
😧 got u

Reply:
done